evm/p2p/peer.py

```diff
@@ -490,8 +514,22 @@ def __init__(self, block_number, block_hash, total_difficulty, genesis_hash):
     try:
         peer = loop.run_until_complete(
             asyncio.wait_for(
-                handshake(remote, ecies.generate_privkey(), LESPeer, chaindb, network_id),
+                handshake(remote, ecies.generate_privkey(), peer_class, chaindb, network_id),
                 HANDSHAKE_TIMEOUT))
+        # Request some stuff, just as a basic test.
+        block_hash = decode_hex(
+            '0x59af08ab31822c992bb3dad92ddb68d820aa4c69e9560f07081fa53f1009b152')
```

Comment:
Assuming this is going to remain beyond just your testing, can you add a comment indicating what block this is

Reply:
This is inside the `if __name__ == "__main__":` block, so it's all just used when testing manually, but I'll add a link to the block this hash belongs to